spaces/domains.json

```
@@ -501,5 +501,6 @@
   "dao.cronosfrens.io": "cronosfrens.io",
   "vote.tresleches.finance": "3Leches.eth",
   "gov.wagumi.xyz": "wagumi.eth",
-  "gov.tethys.finance": "tethysswap.eth"
+  "gov.tethys.finance": "tethysswap.eth",
+  "vote.huckleberry.finance":"huckleberrydex.eth"
```

Please add a space between sub-domain and ENS.
```suggestion
  "vote.huckleberry.finance": "huckleberrydex.eth"
```

---

Ok